rspec-rails.gemspec

```diff
@@ -51,6 +51,11 @@ Gem::Specification.new do |s|
   end
 
   s.add_development_dependency 'cucumber', '~> 1.3.5'
-  s.add_development_dependency 'aruba',    '~> 0.14.12'
+  if ENV['RAILS_VERSION'].gsub(/[^\d\.-]/, '').tr('-', '.') < '4.0'
+    # aruba > 0.9 depends on thor ~> 0.19, while railties < 4.0 depend on thor ~> 0.14.6
+    s.add_development_dependency 'aruba', '~> 0.5.4'
```

Comment:
So why not use aruba 0.8?

Reply:
We [used to use](https://github.com/rspec/rspec-rails/pull/2140/files#diff-73c25cf975ccb4c89d8659202506c353L58) 0.5.4 before the update. Tried 0.8 and there were some failures here and there. Decided to stick with what used to work.

Reply:
Do want to update the comment to that effect?

Reply:
Do you think a comment is really necessary?
It's in the blame https://github.com/rspec/rspec-rails/blame/a8dd5e1764b7142e67dc7903842f332e1a4a7daa/rspec-rails.gemspec#L56 https://github.com/rspec/rspec-rails/commit/5781ea1be5c474d6cd47bb2a9d12d106044c7e57

>  Use the previous aruba for older Rails

> https://github.com/rspec/rspec-rails/pull/2140/files#diff-73c25cf975ccb4c89d8659202506c353L58 bumped aruba from 0.5.4 straight to 0.14.12. Using the version that used to work for older rails.

Reply:
The comment above it is misleading currently.